src/bin/faf-migrate-db

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
```

Comment:
In Fedora, we prefer /usr/bin/python3 in shebang.

Reply:
Changed it. Looks like we’ve been sedding these out in the spec file, but I just ripped that out and modified the files directly.